src/translator.js

```diff
@@ -94,6 +94,9 @@ class Translator {
       );
 
       text = this._getValueFromJSON(key, fallbackTranslation, false);
+    } else if (!text) {
+      text = key;
```

Comment:
`text` will be the `key` if no translation is found and if the key also does not exist inside the `defaultLanguage`